soc/sophgo/cv180x/Kconfig.defconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023-2024 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_CV180X
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 25000000
```

Comment:
this this needs to come from DTS

Reply:
@gmarull

Reply:
Thank you for the suggestion. I need some time to investigate the clock tree to understand why the value here (from the official FreeRTOS SDK) is used and why it differs from the CPU frequency stated in the datasheet.